### PR TITLE
fixed sizing of friends list to fit suggested books

### PR DESCRIPTION
**Pull request by maraquepo**
Screen Shot 2022-02-03 at 12 30 32 PM

Fixed sizing of friends list to fit suggested books component

### CODE REVIEW

**Comment by maraquepo** [comment]
Screen Shot 2022-02-03 at 12 40 37 PM

How it looks like w/o friends

File: client/src/components/RightComponent/FriendsList/FriendsList.jsx [hunk 1]
```diff
@@ -13,114 +13,120 @@ import {
...
-    .then((res) => {
-      const newFriends = friendsList.filter((friend) => friend !== friendToRemove);
-      setFriendsList(newFriends);
-    })
-    .catch(err => console.error(err))
+      .then((res) => {
+        const newFriends = friendsList.filter((friend) => friend !== friendToRemove);
+        setFriendsList(newFriends);
+      })
+      .catch((err) => console.error(err));
```
Comment (brynrmrzz) [inline client/src/components/RightComponent/FriendsList/FriendsList.jsx:80]: Just curious is this is a preference is a standard

File: client/src/components/RightComponent/FriendsList/FriendsList.jsx [hunk 2]
```diff
@@ -13,114 +13,120 @@ import {
...
+          >
+            Friends List
+          </Button>
+        </FriendsListButton>
+        <FriendsListContent>
+          {/* <h5 style={{position:'absolute', top: 5, marginBottom: 15}}>Friends Lists</h5> */}
+          {friendsList.length === 0 ? (
+            <div style={{paddingTop: 80}}>
+              {' '}
+              No friends :({' '}
```
Comment (brynrmrzz) [inline client/src/components/RightComponent/FriendsList/FriendsList.jsx:102]: is there a reason for the empty brackets, haven't seen this before.

**Review by brynrmrzz — APPROVED** [review]

File: client/src/components/RightComponent/FriendsList/FriendsList.jsx (hunk 2 above)
Comment (maraquepo) [reply client/src/components/RightComponent/FriendsList/FriendsList.jsx:102]: I think it gets automatically added when I use prettier to format the document sometimes. I can remove it on the next pr. As for the other comment, I didn't add that particular part, but it seems like the standard to be.

**Pull request merged**